src/Scanner.js

```diff
@@ -214,6 +220,13 @@ class DefaultScanner {
         return this.mode === 'stopped';
     }
 
+    /**
+     * @returns {HTMLFormElement|null}
+     */
+    get forcedForm() {
+        return this.device.settings.siteSpecificFeature?.getForcedForm() ?? null;
```

Comment:
I really dislike how we're always forcing to null with the nullish coaleshing operator even when the API naturally returns null, like when using `querySelector`. We're doing it over and over. I'm going to flag them all, bear with me. 🙂